app/App.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <v-app>
     <v-content>
-      <Leaderboard :challenge-id="2019" />
+      <Leaderboard :challenge-id="'2019'" />
```

Comment:
This could be passed as `challenge-id="2019"` if it's a string. `v-bind` is only necessary for non-string types.